### DISCUSSION TITLE
Notification of new releases

### DISCUSSION THREAD

**Discussion by kikeenrique** [variar/klogg#304] [General]
Hi!
I'm subscribed to notifications of new releases, but I'm getting lots of notifications related to Pre-release $flavour tags. 

Is there any way (easy for you) to modify this behaviour?
Would it be possible to use those tags only on "tags" section but not on "releases"?

Thanks a lot for your effort!

**Comment by variar** [comment]
Hi,

There was an issue with lots of notifications from CI releases -- #215. As far as I understand releases with "pre-release" tag should not generate notifications. Can you show a screenshot of notifications that you are receiving?

**Comment by kikeenrique** [comment]
Hi again,

Here is a screenshot with my GitHub config:
<img width="297" alt="Screenshot 2021-05-10 at 21 46 07" src="https://user-images.githubusercontent.com/2754671/117715872-27484780-b1d9-11eb-9aea-4814bcbeb759.png">
And one with the emails received today.
<img width="576" alt="Screenshot 2021-05-10 at 21 44 14" src="https://user-images.githubusercontent.com/2754671/117715758-02ec6b00-b1d9-11eb-8418-f3b9f536cfef.png">

I've been reading [GitHub releases documentation](https://docs.github.com/en/github/administering-a-repository/automation-for-release-forms-with-query-parameters) and I understand that prerelease creates a release, just adding a special flag that will be identified as non-production ready.
Looking at next screenshot from the releases sections, it's what I would expect based on documentation.
<img width="764" alt="Screenshot 2021-05-10 at 21 55 14" src="https://user-images.githubusercontent.com/2754671/117717097-a722e180-b1da-11eb-9b4a-fd8e2a976455.png">

**Reply by variar** [reply]
Well, maybe I should not push to master branch so often :)

Joking aside I'll try to mark thess pre-releases as "drafts". Let me know if notifications are still coming.

**Reply by variar** [reply]
Nope, didn't work as expected. Marking releases as drafts produces links that are visible only for maintainers. Leaving pre-releases only in "tags" section is not possible, because there is no way to add assets to tags. And whole point of pre-releases is to provide precompiled packages.

**Comment by kikeenrique** [comment]
I see.
Then I think I will rely on klogg update check instead of GitHub releases notification. 
At least until GitHub allows filtering pre-releases, if that ever occurs.
Thanks for your help in any case!

**Reply by variar** [reply]
Try open a discussion in GitHub [feedback](https://github.com/github/feedback/discussions). Filtering out pre-releases is quite good feature. I think many people would like to have it.

**Comment by variar** [comment]
Looks like it is not possible not to create releases for CI builds, because assets can't be added to tags. Klogg has built-in version checking that from time to time checks https://github.com/variar/klogg/blob/master/latest.json and shows pop-up dialog when new version  is available. For stable builds it checks only for new stable versions. And if some build from CI is used, it checks for new CI version mentioned in that file. I mark CI version as stable about once a week, so there should be not many such notifications.

**Comment by variar** [comment]
I've modified workflow in such a way that CI releases should not be created on each push to master branch. Full build with new pre-release can be triggered either manually or by putting a special tag in commit message.
